### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by bendichter**
This should fix our license shield in the README

### CODE REVIEW

**Review by CodyCBakerPhD — DISMISSED** [review]

File: setup.py [hunk 1]
```diff
@@ -53,6 +53,7 @@
     author_email="ben.dichter@catalystneuro.com",
     url="https://github.com/catalystneuro/neuroconv",
     keywords="nwb",
+    license_files=('LICENSE.txt',)
```
Comment (bendichter) [inline setup.py:56]:
```suggestion
    license_files=('LICENSE',)
```

**Review by CodyCBakerPhD — DISMISSED** [review]

File: setup.py [hunk 2]
```diff
@@ -53,6 +53,7 @@
     author_email="ben.dichter@catalystneuro.com",
     url="https://github.com/catalystneuro/neuroconv",
     keywords="nwb",
+    license_files=('LICENSE',)
```
Comment (CodyCBakerPhD) [inline setup.py:56]:
```suggestion
    license_files=('LICENSE')
```

Comment (bendichter) [inline setup.py:56]:
```suggestion
    license_files=('LICENSE',),
```

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by bendichter** [comment]
@CodyCBakerPhD maxwell is causing problems again :-/

**Comment by CodyCBakerPhD** [comment]
> @CodyCBakerPhD maxwell is causing problems again :-/

Yeah, I’ll have a fix for it later - relying on the law of averages for now

**Pull request merged**